cpp/src/Ice/Incoming.cpp

```diff
@@ -657,45 +617,65 @@ IceInternal::Incoming::invoke(const ServantManagerPtr& servantManager, InputStre
         }
     }
 
+    const bool amd = false;
```

Comment:
I find this very confusing and this raise the question of whether or not we should do this for all boolean parameters... It's really a C++ deficiency.

Reply:
Which style to you suggest?
```
const bool amd = false;
response(amd);

vs

response(false);

vs 

response(/*amd*/ false);

vs

response(false); // amd: false
```
?

Reply:
switched to:
```
response(false); // amd: false
```

Reply:
BTW this amd parameter is only used for collocated calls. It's not clear to me why we need it.